### DISCUSSION TITLE
Do I need ... require-dev entries

### DISCUSSION THREAD

**Discussion by kopieMK** [pimcore/pimcore#14632] [General]
Hi folks!

In our composer.json for pimcore we have some entries under "require-dev". 

```
	"require-dev": {
		"cache/integration-tests": "^0.17.0",
		"codeception/codeception": "^4.1.12",
		"codeception/module-symfony": "^1",
	},
```
I compared with the current composer.json from the official demo project. There is nothing like this noted. I have a quick look at codeception, which is for testing (... right?). My question, do the composer.json/Pimcore need this entries in the production env/ live system?

Thank you so much for any feedback!

Greetings

Magnus

**Comment by SamyMP** [comment]
Hi, 

You don't need the packages in the "require-dev" section of your `composer.json` in you production environment, they are only installed in dev to test your application.

When installing/updating pimcore on your prod environment, make sure to add the `--no-dev` flag on your composer command

```bash
composer install --no-dev
```

**Reply by kopieMK** [reply]
Ok, thank you so much! I had hoped that it is so.
